assets/javascript/apps/pipeline/stores/pipelineManagerStore.ts

```diff
@@ -12,11 +12,14 @@ const usePipelineManagerStore = create<PipelineManagerStoreType>((set, get) => (
   dirty: false,
   isSaving: false,
   isLoading: true,
+  errors: {},
   loadPipeline: async (pipelineId: number) => {
     set({isLoading: true});
     apiClient.getPipeline(pipelineId).then((pipeline) => {
       if (pipeline) {
         set({currentPipeline: pipeline, currentPipelineId: pipelineId});
+        // TODO: Remove this below and read erros from the ReactFlowJsonObject
```

Comment:
TODO?

Reply:
Oh yeah that is not relevant anymore. I'm keeping the errors on this store level. I'll remove this